Package.swift

```diff
@@ -150,12 +150,12 @@ let package = Package(
     .package(
       name: "GoogleUtilities",
       url: "https://github.com/google/GoogleUtilities.git",
-      "7.2.1" ..< "8.0.0"
+      "7.3.0" ..< "8.0.0"
```

Comment:
Double checking, should this be 7.3 or 7.4?

Reply:
Hmm is 7.3 the latest minor that we bumped to?

Reply:
From the PR description - this needs to wait until we publicly release GoogleUtilities on SwiftPM.

Reply:
oof 🤦 , sorry... I understand now!